book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :author, :title
+  attr_reader :rentals
```

Comment:
Please add both getters and setters for `@rentals` in the `Book` class.

Reply:
Thank you